.github/workflows/installations.yml

```diff
@@ -10,8 +10,9 @@ on:
     - '.github/workflows/installations.yml'
     - 'Gemfile'
   schedule:
-    # Run every day at 11pm (PST) - cron uses UTC times
-    - cron:  '0 7 * * *'
+    # Run every day at 3am (PST) - cron uses UTC times
+    # This is set to 3 hours after zip workflow so zip testing can run after.
```

Comment:
This is set to 3 hours after zip workflow finishes so zip testing can run after.